src/OrleansAzureUtils/Storage/AzureStorageUtils.cs

```diff
@@ -275,10 +275,10 @@ public static string SanitizeTableProperty(string key)
         {
             // Remove any characters that can't be used in Azure PartitionKey or RowKey values
             // http://www.jamestharpe.com/web-development/azure-table-service-character-combinations-disallowed-in-partitionkey-rowkey/
-            key.Replace('/', '_');  // Forward slash
-            key.Replace('\\', '_'); // Backslash
-            key.Replace('#', '_');  // Pound sign
-            key.Replace('?', '_');  // Question mark
+            key = key.Replace('/', '_').  // Forward slash
+                Replace('\\', '_').       // Backslash
+                Replace('#', '_').        // Pound sign
+                Replace('?', '_');        // Question mark
```

Comment:
BTW, feel free to push back on the comment if it's going to require a lot of context switching or fighting git from you, this is a VERY minor comment that I don't feel *absolutely must* be addressed :tongue: